src/routes/index.js

```diff
@@ -18,6 +19,11 @@ const routes = [
     path: routesPaths.login,
     component: <Login />,
   },
+  {
+    path: routesPaths.profile,
```

Comment:
Just a side comment, recently I discovered react-router-dom allows for nesting pages (see [here](https://v5.reactrouter.com/web/example/nesting) )
This makes this approach unnecessary. I think it's okay to leave it like this, but JFYI